content/scripts/build.js

```diff
@@ -253,20 +274,29 @@ class Builder {
         })
       : null;
   }
+
   initProgressbar(total) {
     this.progressBar && this.progressBar.init(total);
   }
+
   tickProgressbar(incr) {
     this.progressBar && this.progressBar.update(incr);
   }
-  stopProgressbar() {
-    this.progressBar && this.progressBar.stop();
-  }
```

Comment:
I don't understand this. How come we don't need that? `this.progressBar.stop();` sends that last final `\n` to the console. No?

Reply:
I removed it for two reasons:
- We never actually called `stopProgressbar()`.
- I changed the progress bar such that it "stops" automatically when it reaches 100%.
but I'm happy to change this approach as desired.

Reply:
Oh, I see that you caught that in one of your comments below.

Reply:
That actually explains some bugs we've had with the progressbar! Thanks!
However, it's kinda nice that here, in this region of the code, there's a `initProgressbar`, `tickProgressbar`. So that code that you put in that compares `current` with `total` perhaps it can call `this.stopProgressbar`. Nit.